olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogRequest.h

```diff
@@ -33,26 +33,31 @@ namespace dataservice {
 namespace read {
 
 /**
- * @brief The CatalogRequest class encapsulates the fields required to request
- * the Catalog configuration.
+ * @brief Encapsulates the fields required to request catalog configuration.
  */
 class DATASERVICE_READ_API CatalogRequest final {
  public:
   /**
-   * @brief BillingTag is an optional free-form tag which is used for
-   * grouping billing records together. If supplied, it must be between 4 - 16
-   * characters, contain only alpha/numeric ASCII characters  [A-Za-z0-9].
-   * @return the billing tag, or boost::none if not set.
+   * @brief Gets a billing tag to group billing records together.
+   *
+   * The billing tag is an optional free-form tag that is used for grouping
+   * billing records together. If supplied, it must be 4&ndash;16 characters
```

Comment:
```suggestion
   * billing records together. If supplied, it must be 4-16 characters
```
Would a real dash, i.e. `-`, be cleaner? The encoding is nice for HTML but it's harsh on humans:)

Reply:
Agree.
Added the en dash sign (–) instead of `&ndash;`.